src/acquisition/covidcast/covidcast_meta_cache_updater.py

```diff
@@ -3,30 +3,38 @@
 # standard library
 import argparse
 import sys
+import time
 
 # first party
 from delphi.epidata.acquisition.covidcast.database import Database
+from delphi.epidata.acquisition.covidcast.logger import get_structured_logger
 from delphi.epidata.client.delphi_epidata import Epidata
 
-
 def get_argument_parser():
   """Define command line arguments."""
 
-  # there are no flags, but --help will still work
-  return argparse.ArgumentParser()
+  parser = argparse.ArgumentParser()
+  parser.add_argument("--log_file", help="filename for log output")
+  return parser
 
 
 def main(args, epidata_impl=Epidata, database_impl=Database):
   """Update the covidcast metadata cache.
 
   `args`: parsed command-line arguments
   """
+  logger = get_structured_logger(
+      "metadata_cache_updater",
+      filename=args.log_file)
```

Comment:
i take it this means the log file argument is required...  is there maybe a way to let it default to stdout or somehing?

Reply:
If the arg is empty/none then the logger will default to stdout.